lib/tasks/usps_enrollment.rake

```diff
@@ -0,0 +1,71 @@
+namespace :usps_enrollment do
+
+  desc 'Pass an enrollment'
+  task pass: :environment do
+    # FIXME: Check if we're in prod, and yell at the user + fail if so!
+
+    enrollment = InPersonEnrollment.find_by_enrollment_code(ENV.fetch('ENROLLMENT_CODE', ''))
+    raise "No such enrollment" unless enrollment.present?
+
+    # Do we still need this?
+    enrollment.update(unique_id: enrollment.usps_unique_id) if enrollment.unique_id.blank?
+
+    mock_response = {
+      # TODO: This should actually be in CST apparently.
+      'transactionEndDateTime' => Time.zone.now.strftime('%m/%d/%Y %H%M%S'),
+      'fromRakeTask' => 'yes'
```

Comment:
This has no specific purpose yet, but I want to start laying the groundwork for making it clear where these results came from.